fableous-fe/src/Api.tsx

```diff
@@ -11,7 +11,7 @@ const baseWS =
     ? `${process.env.REACT_APP_BACKENDWS}`
     : `wss://${window.location.hostname}`;
 
-const apiClient = axios.create({
+export const apiClient = axios.create({
```

Comment:
Don't need to export this?

Reply:
👽

Reply:
yup my bad, should add axios.defaults.baseURL = baseAPI;

remove the export and apiClient could just be axios.create()